### PR TITLE
[DowngradePhp80] Handle DowngradeMixedTypeDeclarationRector on Closure

### PR DESCRIPTION
**Pull request by samsonasik**
ref https://github.com/rectorphp/rector-src/pull/105#issuecomment-848380283

### CODE REVIEW

**Comment by samsonasik** [comment]
All checks have passed 🎉 @TomasVotruba it is ready for review.

File: rules/DowngradePhp80/Rector/FunctionLike/DowngradeMixedTypeDeclarationRector.php [hunk 1]
```diff
@@ -28,7 +29,7 @@ public function __construct(
      */
     public function getNodeTypes(): array
     {
-        return [Function_::class, ClassMethod::class];
+        return [Function_::class, ClassMethod::class, Closure::class];
```
Comment (TomasVotruba) [inline rules/DowngradePhp80/Rector/FunctionLike/DowngradeMixedTypeDeclarationRector.php:32]: How simple fix :clap:

**Comment by TomasVotruba** [comment]
Thank you :+1:

**Pull request merged**